components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -448,7 +450,8 @@ export class GithubApp {
         const contextURL = pr.html_url;
 
         const isFork = pr.head.repo.id !== pr.base.repo.id;
-        const runPrebuild = this.appRules.shouldRunPrebuild(config, false, true, isFork);
+        const runPrebuild =
```

Comment:
<img width="812" alt="Screen Shot 2022-05-20 at 15 50 09" src="https://user-images.githubusercontent.com/914497/169542664-12a54b01-b985-4154-b8d9-b12d43077ce7.png">
<img width="909" alt="Screen Shot 2022-05-20 at 15 50 21" src="https://user-images.githubusercontent.com/914497/169542674-26f100f8-a204-4759-a150-0695a686e35e.png">



> ... no prebuild tasks configured

that's looks a bit redundant in two ways.

Reply:
True. Maybe I remove the check from `appRules`? (Different concerns: Prebuild Manager wants an explicit config, with prebuild tasks, and `appRules` wants a `github.prebuild` config that matches whatever branch/pr the webhook is for?)

Reply:
Fixed:

<img width="1164" alt="Screenshot 2022-05-20 at 16 07 29" src="https://user-images.githubusercontent.com/599268/169545704-e5aa87d7-bda3-4403-b0c3-a320475cb3ad.png">